web/styles/search.css

```diff
@@ -100,7 +100,7 @@
             }
         }
 
-        @media (width < $sm_min) {
+        @media (height < 600px) {
```

Comment:
@timabbott it would be nice to put this size in a variable, but the `media_breakpoints` are currently all related to height and there aren't other examples of media queries on height in the codebase, so I wanted to ask first about a good way to set up the naming.

Reply:
Probably a separate area of the CSS variables file, with maybe a `$height_` prefix? These should be pretty rare, I'd think, but no harm in paving the way for more of them.

Reply:
That plan works for me.